modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -628,6 +657,56 @@ public void awaitStarted() throws IgniteCheckedException {
         U.await(cacheStartedLatch);
     }
 
+    /**
+     * Starts the asynchronous operation of pending tasks execution. Is called on start.
+     */
+    private void asyncLocalContinuousTasksExecution() {
+        assert localContinuousTasks != null;
+
+        Set<LocalContinuousTask> tasks = new HashSet<>(localContinuousTasks.values());
```

Comment:
up to you, but may be compactify ? 
localContinuousTasks.values().forEach(t -> asyncLocalContinuousTaskExecute(t, true));